### ISSUE TITLE
Focus behaviour when setting the password

### ISSUE THREAD

**Issue by ed255** [iden3/Wallet#38]
Current: The tab button in the new password prompt brings the focus to the eye icon, instead of bringing the focus to the next password field.

Expected: The tab button should jump from one password field to the other.

@cjgarde says: LP: All related to A11y is "work in progress", to be fixed in the future

**Comment by cjgarde** [comment]
Yes! I change it to a11y label. Because we need to work a lot in this!
thanks!
